public/plugin.ts

```diff
@@ -74,7 +75,7 @@ export class OpendistroSecurityPlugin
       core.application.register({
         id: PLUGIN_NAME,
         title: 'Security',
-        order: 1,
+        order: 9040,
```

Comment:
could we try 
```
        order:DEFAULT_APP_CATEGORIES.management+40
```

we might apply same conversation to all other odfe plugins as well

Reply:
If I do DEFAULT_APP_CATEGORIES.management.order+40 that is equal to 5040, in that case "Security" comes above all other Management plugins. I can do like this though  `order: DEFAULT_APP_CATEGORIES.management.order + 4040`

Reply:
is this number 9040 come from order of management plugin
if so, would you adjust to a different number like 9050 or 9100 to avoid collision

https://github.com/elastic/kibana/blob/master/src/plugins/management/public/plugin.ts

Reply:
@seraphjiang sure, I will make it to 9050 then.